client/src/app/pages/environment-deployment-history/environment-deployment-history.component.html

```diff
@@ -17,42 +34,66 @@
         </tr>
       </ng-template>
     </p-table>
-  } @else if (deploymentsQuery.data()?.length === 0) {
+  } @else if (activityHistoryQuery.data()?.length === 0) {
     <!-- Empty State -->
-    <div class="text-gray-500">No deployments found.</div>
+    <div class="text-gray-500">No deployments and locking found.</div>
```

Comment:
nitpiclk (can also be done in a separate pr): we should use similar emptystates and use the template of the table for it.
See here (from the branches-table.component.html):
```html
<ng-template pTemplate="emptymessage">
      <tr>
        <td colspan="4">
          <div class="flex flex-col gap-2 p-20 justify-center items-center">
            <i-tabler name="git-branch" class="!h-20 !w-20 text-red-500" />
            <span class="font-semibold text-xl">There are no branches found matching your filter criteria.</span>
            <span>Try clearing the filter or reloading the page.</span>
            <p-button (click)="searchTableService.clearFilter(table)">Clear Filter</p-button>
          </div>
        </td>
      </tr>
    </ng-template>
```